packages/kit/src/core/create_manifest_data/index.js

```diff
@@ -238,9 +239,38 @@ export default function create_manifest_data({ config, output, cwd = process.cwd
 	walk(config.kit.files.routes, [], [], [layout], [error]);
 
 	const assets_dir = config.kit.files.assets;
+	/**
+	 * @type {import('types/internal').Asset[]}
+	 */
+	let assets = [];
+	if (fs.existsSync(assets_dir)) {
+		/**
+		 * @type {string[]}
+		 */
+		let exclusions = config.kit.serviceWorker.filesExclusions || [];
+
+		// .DS_STORE files are automatically removed to keep the compatiblity
+		exclusions = [...exclusions, '**/.DS_STORE'];
```

Comment:
@ignatiusmb Hey. What would be a better place then?
I added it here because when using an IDE(as a user of SvelteKit) you go to the definition of `serviceWorker.exclude`, it would immediately let you know what this option is for. 

I have removed the comment from there as per your suggestion but would like to know where the comment should go.

Reply:
Done